ts/sagas/startup.ts

```diff
@@ -132,6 +134,13 @@ function* initializeApplicationSaga(): IterableIterator<Effect> {
   // User is autenticated, session token is valid
   //
 
+  const walletToken: Option<string> = yield select(walletTokenSelector);
+  if (walletToken.isSome()) {
+    // the wallet token is available,
+    // proceed with starting the "watch wallet" saga
+    yield fork(watchWalletSaga, walletToken.value);
```

Comment:
it would be awesome (and much easier to test) if you could pass an instance of the PagoPa client initialized with the token, as we do for the `BackendClient` - in this way completely decouple the wallet saga from the API client